widgetssdk/src/main/java/com/glia/widgets/callvisualizer/EndScreenSharingActivity.kt

```diff
@@ -7,6 +7,14 @@ import com.glia.widgets.di.Dependencies
 import com.glia.widgets.helper.Logger
 import com.glia.widgets.helper.TAG
 
+/**
+ * Glia internal class.
+ *
+ * Should be automatically added to integrator Manifest by Manifest merger during compilation.
```

Comment:
For all these classes:

Wording is a little off.

"Should be automatically added...", I don't think should is the correct wording. "Will be automatically added..." would be better

Reply:
Fixed in https://github.com/salemove/android-sdk-widgets/compare/6d99e10a5b0dfb8e2a59711761de88408328caba..a6dc35614a60fde9f1d0ad33dd7c765644200386